docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-integration.md

```diff
@@ -40,10 +40,12 @@ Create an integration configuration.
 | Flag | Description |
 | :--- | :--- |
 | --config-name | Required. Name for the integration configuration. |
-| --type | Required. The third party tool to export to. Options: DATADOG, GRAFANA, SUMOLOGIC. |
+| --type | Required. The third party tool to export to. Options: DATADOG, GRAFANA, SUMOLOGIC, VICTORIAMETRICS, PROMETHEUS. |
 | --datadog-spec | Required for type DATADOG. The Datadog export details, provided as key-value pairs.<br>Arguments:<ul><li>api-key - your Datadog API key.</li><li>site - your Datadog site parameters.</li></ul> |
 | --grafana-spec | Required for type GRAFANA. The Grafana Cloud export details, provided as key-value pairs.<br>Arguments:<ul><li>access-policy-token - your Grafana Cloud token.</li><li>org-slug - your organization name.</li><li>instance-id - your Grafana Cloud instance ID.</li><li>zone - your Grafana Cloud instance zone.</li></ul> |
 | --sumologic-spec | Required for type SUMOLOGIC. The Sumo Logic export details, provided as key-value pairs.<br>Arguments:<ul><li>access-key - your Sumo Logic access key.</li><li>access-id - your Sumo Logic access ID.</li><li>installation-token - your Sumo Logic installation token.</li></ul> |
+| --prometheus-spec | Required for type PROMETHEUS. The Prometheus export details, provided as key-value pairs.<br>Arguments:<br><ul><li>`endpoint` - your Prometheus OTLP endpoint URL.</li></ul>|
+| --victoriametrics-spec | Required for type VICTORIAMETRICS. The VictoriaMetrics export details, provided as key-value pairs.<br>Arguments:<br><ul><li>`endpoint` - your VictoriaMetrics OTLP endpoint URL.</li></ul>|
```

Comment:
```suggestion
| &#8209;&#8209;victoriametrics&#8209;spec | Required for type VICTORIAMETRICS. The VictoriaMetrics export details, provided as key-value pairs.<br>Arguments:<br><ul><li>`endpoint` - your VictoriaMetrics OTLP endpoint URL.</li></ul>|
```

Reply:
Is it necessary? I don't see it written that way in other flags.

Reply:
These are non-breaking hyphens just so that the flag names don't break - it looks nicer - but you only need to do it on the longest flag name in the table

Reply:
Okay, understood. Made the changes